README.md

```diff
@@ -4,31 +4,123 @@
 [![Build Status][circle-image]][circle-url]
 [![Known Vulnerabilities][snyk-image]][snyk-url]
 
-**WIP** [pprof][pprof-url] support for Node.js.
+[pprof][pprof-url] support for Node.js.
 
-# Running the system test
-The system test starts a simple benchmark, uses this module to collect a time
-and a heap profile, and then verifies that the profiles can be opened and that
-the profiles contain functions from within the benchmark. 
+## Prerequisites
+1. Your application will need to be using Node.js 8.9.4 or greater, or Node.js 
+10.4.1 or greater. The profiler will not be enabled when using earlier versions
+of Node 8 and 10 because the profiler is not stable with those versions of
+Node.js.
+    * Versions of Node.js 8 prior to 8.9.4 are impacted by
+    [this](https://bugs.chromium.org/p/v8/issues/detail?id=6623) issue, which
+    causes a memory leak when time profiling is enabled.
+    * Versions of Node.js 10 prior to 10.4.1 are impacted by
+    [this](https://bugs.chromium.org/p/chromium/issues/detail?id=847863) issue,
+    which can cause garbage collection to take several minutes when heap
+    profiling is enabled.
 
-To run the system test, [golang](https://golang.org/) must be installed.
+2. The `pprof` module has a native component that is used to collect profiles 
+with v8's CPU and Heap profilers. You may need to install additional
+dependencies to build this module.
+    * For Linux: `pprof` has prebuilt binaries available for Linux and Alpine
+    Linux for Node 8, 10, and 11. No additional dependencies are required.
+    * For other environments: when using `@google-cloud/profiler` on environments
+    that `pprof` does not have prebuilt binaries for, the module
+    [`node-gyp`](https://www.npmjs.com/package/node-gyp) will be used to
+    build binaries. See `node-gyp`'s
+    [documentation](https://github.com/nodejs/node-gyp#installation)
+    for information on dependencies required to build binaries with `node-gyp`.
 
-The following command can be used to run the system test with all supported
-versions of Node.JS:
-```console
-$ sh system-test/system_test.sh
-```
+3. The [`pprof`][pprof-url] CLI can be used to view profiles collected with 
+this module. Instructions for installing the `pprof` CLI can be found
+[here][pprof-install-url].
 
-To run the system test with the v8 canary build, use:
-```console
-$ RUN_ONLY_V8_CANARY_TEST=true sh system-test/system_test.sh
-```
+## Basic Set-up
+
+Install [`pprof`][npm-url] with `npm` or add to your `package.json`.
+
+  ```sh
```

Comment:
Between the two files, there is mixed usage of 'sh' and 'console', with and without the $ sign. 
Making the shell commands consistent would be nice.

Reply:
Done.